assets/src/components/map.tsx

```diff
@@ -164,13 +164,10 @@ const Shape = ({ shape }: { shape: Shape }) => {
 
 export const autoCenter = (
   map: LeafletMap,
-  vehicles: Vehicle[],
+  latLngs: LatLngExpression[],
```

Comment:
Maybe `vehicleLatLngs` would be a little more descriptive?

Reply:
At this point, you don't really care what the latlngs are of. Maybe someday we'd decide that any stops should be in view, too, and you could passing `autoCenter(vehicleLatLngs.concat(stopLatLngs))`. This function wouldn't change.

Reply:
Ok, sure makes sense.